cmd/backendcmd/spawn_server.go

```diff
@@ -1,15 +1,18 @@
 // Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
-package cmd
+package backendcmd
```

Comment:
I'm not sure about this cmd postfix. It is already inside cmd directory. Probably you can move other stuff to pkg (prompts, mocks), and only keep backend, network, subnet.

Reply:
It's probably a good idea to move those packages, but I do think we should keep the `-cmd` suffix. We have several packages with similar names (ex: `pkg/subnet` and `cmd/subnet`). I think we should name them distinctly to avoid confusion.